src/background/hypothesis-chrome-extension.js

```diff
@@ -210,12 +225,23 @@ export default function HypothesisChromeExtension({
     });
   }
 
+  /**
+   *
+   * @param {chrome.tabs.Tab} tab
+   */
```

Comment:
Small stylistic request - if a function has no description but just param docs then keep avoid unnecessary empty lines in the JSDoc:

```js
/**
 * @param {chrome.tabs.Tab} tab
 */
```

Or even for the common case of a local function with one argument:

```js
/** @param {chrome.tabs.Tab} tab */
```